app/command_shortcuts.go

```diff
@@ -37,7 +37,7 @@ func (me *ShortcutsProvider) GetCommand(T goi18n.TranslateFunc) *model.Command {
 }
 
 func (me *ShortcutsProvider) DoCommand(args *model.CommandArgs, message string) *model.CommandResponse {
-	shortcutIds := [28]string{
+	shortcutIds := [29]string{
```

Comment:
Not something to change now, but we can change this to `[...]string{` at some point to have it autosize so we don't need to keep changing it ourselves :)